### DISCUSSION TITLE
Is it possible to do continuous batching with an openai ChatCompletion compatible interface?

### DISCUSSION THREAD

**Discussion by msugiyama57** [vllm-project/vllm#1605] [Q&A]
The following code shows an example of doing Continuous batching, can it be done in oepnai ChatCompletion format?
https://github.com/vllm-project/vllm/blob/main/examples/offline_inference.py

**Comment by simon-mo** [comment]
The OpenAI server automatically batches concurrent requests already, just try it with concurrent requests using any OpenAI compatible clients!
